content/src/scripts/containers/TreeList/TreeList.jsx

```diff
@@ -80,6 +81,16 @@ function TreeList({
     return firstPageLoad;
   };
 
+  const buildFilterMap = () => {
+    let map = new Map();
+    map['test'] = document.getElementById('filterTests').checked;
+    map['removed'] = document.getElementById('filterRemoved').checked;
+    map['renamed'] = document.getElementById('filterRenamed').checked;
+    map['imports'] = document.getElementById('filteredImports').checked;
+    map['newFile'] = document.getElementById('filteredNewFiles').checked;
+    return map;
```

Comment:
this 'backend' logic works as soon as the checkboxes are checked and the state of these is fixed properly